databricks/koalas/series.py

```diff
@@ -354,9 +364,9 @@ def _unpack_scalar(sdf):
     Takes a dataframe that is supposed to contain a single row with a single scalar value,
     and returns this value.
     """
-    l = sdf.head(2)
-    assert len(l) == 1, (sdf, l)
-    row = l[0]
+    ret = sdf.head(1)
```

Comment:
Actually I thought this was because we wanted to check the sdf returns exactly one row.

Btw, we have one more similar place in `frame.py`.

Reply:
Hmm .. let me revert this change back for now.